string board
project/Point[] pieces
uint8 flag
string error